setup.py

```diff
@@ -156,6 +155,25 @@ def gen_packages_items():
             'build': parse_requirements('requirements/build.txt'),
             'optional': parse_requirements('requirements/optional.txt'),
         },
-        ext_modules=[],
+        ext_modules=[
+            make_cuda_ext(
+                name='nms_ext',
+                module='mmdet.ops.nms',
+                sources=['src/nms_ext.cpp', 'src/cpu/nms_cpu.cpp'],
+                sources_cuda=[
+                    'src/cuda/nms_cuda.cpp', 'src/cuda/nms_kernel.cu'
+                ]),
+            make_cuda_ext(
+                name='roi_align_ext',
+                module='mmdet.ops.roi_align',
+                sources=[
+                    'src/roi_align_ext.cpp',
+                    'src/cpu/roi_align_v2.cpp',
+                ],
+                sources_cuda=[
+                    'src/cuda/roi_align_kernel.cu',
+                    'src/cuda/roi_align_kernel_v2.cu'
+                ]),
+        ],
```

Comment:
Let's get rid of these duplicated and move to mmcv.

Reply:
Suggest to remain it as is now and do it later.

Reply:
Why? What makes it being out of this PR's scope?

Reply:
As I remember with extensions implemented in `mmcv` our python code does not work (only deformable convolutions work). I will check it again.

Reply:
Local `RoIAlign` was removed.